.copier-answers.yml

```diff
@@ -0,0 +1,14 @@
+# Changes here will be overwritten by Copier; NEVER EDIT MANUALLY
+_commit: v4.3.5
+_src_path: https://github.com/jupyterlab/extension-template
+author_email: jupyter@googlegroups.com
+author_name: Jupyter Development Team
+has_binder: true
+has_settings: true
+kind: server
+labextension_name: jupyterlab_latex
```

Comment:
This does not look right. Shouldn't it be `@jupyterlab/latex`?

Reply:
This file is generated by copier and mentions "# Changes here will be overwritten by Copier; NEVER EDIT MANUALLY" 
I am not entirely sure why is mentions the "labextension_name" as the node package name. Will check.

Reply:
Fixed. 
labextension_name typically follows the format `@<organization>/<package-name>` for scoped packages or just `<package-name>` for non-scoped packages.